<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesWindow</class>
 <widget class="QDialog" name="PreferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <item>
        <property name="text">
         <string>Display</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Media Libruary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Playlist</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string>Status</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QCheckBox" name="ShowStatusBar">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Show Status Bar</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_3">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>218</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="showPlplayingTime">
                      <property name="text">
                       <string>Show playing time in playlist</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_4">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>138</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>78</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_4">
                 <property name="title">
                  <string>Display Formatting</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_3">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_3">
                        <item>
                         <widget class="QLabel" name="label">
                          <property name="text">
                           <string>Window Title</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="windowFormat"/>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="verticalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_4">
                        <item>
                         <widget class="QLabel" name="label_2">
                          <property name="text">
                           <string>Status Bar</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_2">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="statusFormat">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_7">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>148</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QGridLayout" name="gridLayout_7"/>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Maximum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSplitter" name="splitter_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>   Music Folder</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="1" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLineEdit" name="musFolder"/>
                     </item>
                     <item>
                      <widget class="QPushButton" name="musFolderChange">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="text">
                        <string>Change</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item row="2" column="0">
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>   File Types</string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="1" column="0">
                    <widget class="QLineEdit" name="fileFilter"/>
                   </item>
                   <item row="2" column="0">
                    <spacer name="verticalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>208</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QListWidget" name="colList">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1" rowspan="5">
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <spacer name="verticalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="title">
                  <string>Column Settings</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_7">
                      <item>
                       <widget class="QLabel" name="label_3">
                        <property name="text">
                         <string>Title</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="colTitle">
                        <property name="minimumSize">
                         <size>
                          <width>200</width>
                          <height>0</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_5">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>40</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_8">
                      <item>
                       <widget class="QLabel" name="label_4">
                        <property name="text">
                         <string>Width</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLineEdit" name="colSize"/>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Display Formating</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPlainTextEdit" name="colFormat"/>
                    </item>
                   </layout>
                  </item>
                  <item row="4" column="0">
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QPushButton" name="colReset">
                      <property name="text">
                       <string>Reset</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <spacer name="horizontalSpacer_6">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>68</width>
                        <height>20</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item>
                     <widget class="QPushButton" name="colSave">
                      <property name="text">
                       <string>Save</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="colApply">
                      <property name="text">
                       <string>Apply</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="7" column="0">
                   <widget class="QPlainTextEdit" name="colPlayFormat"/>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Current playing track formating</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_10">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="5" column="0">
                   <spacer name="verticalSpacer_11">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="3" column="0">
                   <widget class="QPlainTextEdit" name="colBack"/>
                  </item>
                  <item row="8" column="0">
                   <widget class="QPlainTextEdit" name="colPlayBack"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_9">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>New column</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_2">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Move Up</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButton_3">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Move Down</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pushButton_4">
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Remove Column</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
